migration/FromMySqlToPostgreSql/FromMySqlToPostgreSql.php

```diff
@@ -712,7 +712,12 @@ private function arrangeColumnsData(array $arrColumns)
                 || stripos($arrColumn['Type'], 'linestring') !== false
                 || stripos($arrColumn['Type'], 'polygon') !== false
             ) {
-                $strRetVal .= 'HEX(ST_AsWKB(`' . $arrColumn['Field'] . '`)),';
+                if ( strcmp( '5.5', substr( $this->mysqlVersion, 0, 3 ) ) == 0
```

Comment:
I think, that <code>if( mysql <= 5.5 )</code> is much better option.<br>
Could you modify the P.R. a little bit?